Dockerfile.daemon

```diff
@@ -2,6 +2,27 @@ FROM golang:1.18 AS builder
 WORKDIR /go/src/github.com/openshift/ingress-node-firewall
 COPY . .
 
+#Install libxdp and libbpf headers
```

Comment:
Ah I see now why in the README earlier you mentioned how to build stuff.
The golang builder pod is debian based, so we have to compile  from source, then compile go and because it's all statically linked we can copy the binary over to a quay system. That creates a little bit of a dilemma, though, because we are not using a Red Hat supported version of libbpf or libxdp...?

Reply:
With that said, we also have to update Dockerfile.daemon.openshift and there I suppose that we pull in the deps with yum?

Reply:
correct or switch to use fedora builder so we can install those pkgs instead of the hack I have ?

Reply:
Really weird: I can find the deps in brew but not in our rhel repos:
https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2123985

~~~
yum install http://download.eng.bos.redhat.com/brewroot/vol/rhel-8/packages/xdp-tools/1.2.6/1.el8/x86_64/libxdp-1.2.6-1.el8.x86_64.rpm http://download.eng.bos.redhat.com/brewroot/vol/rhel-8/packages/xdp-tools/1.2.6/1.el8/x86_64/libxdp-devel-1.2.6-1.el8.x86_64.rpm http://download.eng.bos.redhat.com/brewroot/vol/rhel-8/packages/xdp-tools/1.2.6/1.el8/x86_64/xdp-tools-1.2.6-1.el8.x86_64.rpm
~~~

Reply:
cc'd @tohojo